aea/aea_builder.py

```diff
@@ -282,7 +282,7 @@ class AEABuilder:
 
     """
 
-    DEFAULT_AGENT_LOOP_TIMEOUT = 0.05
+    DEFAULT_AGENT_ACT_PERIOD = 1
```

Comment:
Why has this been increased so dramatically?

Reply:
cause variable is used for other purposes.
initially, it was used as a delay.
now as a period. and used to update skills.

updated back to 0.05